lib/plan-builder-base.js

```diff
@@ -122,11 +122,11 @@ function castArg(arg, funcName, paramName, argPos, paramTypes) {
         const value = arg[key];
         switch(key) {
           case 'scoreMethod':
-            if (['logtfidf', 'logtf', 'simple', 'score-logtfidf', 'score-logtf', 'score-simple'].includes(value)) {
+            if (['logtfidf', 'logtf', 'simple', 'score-logtfidf', 'score-logtf', 'score-simple', 'zero', 'random', 'bm25'].includes(value)) {
               return true;
             }
             throw new Error(
-                `${argLabel(funcName, paramName, argPos)} can only be logtfidf, logtf, or simple`
+                `${argLabel(funcName, paramName, argPos)} can only be logtfidf, logtf, simple, 'zero', 'random' or bm25`
```

Comment:
I like the use of single quotes around the values, but it should be done for every value.